src/components/Router/StepComponentMap.js

```diff
@@ -44,15 +45,21 @@ const captureStepsComponents = (documentType, mobileFlow, steps, deviceHasCamera
   const complete = mobileFlow ? [ClientSuccess] : [Complete]
   return {
     welcome: () => [Welcome],
-    face: () => shouldUseVideo(steps) ?
-        getRequiredVideoSteps(deviceHasCameraSupport, mobileFlow) :
-        [SelfieCapture, SelfieConfirm],
+    face: () => getFaceSteps(steps, deviceHasCameraSupport, mobileFlow),
     document: () => createIdentityDocumentComponents(documentType, hasPreselectedDocument(steps)),
     poa: () => [PoAIntro, SelectPoADocument, PoAGuidance, PoACapture, DocumentFrontConfirm],
     complete: () => complete
   }
 }
 
+const getFaceSteps = (steps, deviceHasCameraSupport, mobileFlow) => {
+  const faceStep = steps.filter(step => step.type === "face")[0]
+  const useWebcam = faceStep.options.useWebcam === undefined ? true : faceStep.options.useWebcam
```

Comment:
this default should be done at index.js which is where we are formatting steps and assigning high level defaults.

this might be a problem already in other places